### PR TITLE
Feature/conv 10 update GIT flow with data from confluence

### PR DESCRIPTION
**Pull request by 7img**

### CODE REVIEW

File: git/README.md [hunk 1]
```diff
@@ -11,6 +11,11 @@ Prefix commit message with a [JIRA](https://www.atlassian.com/software/jira) ID
 - JIRA-123 Add Git conventions
 - Add Git conventions (JIRA-123)
 
+## Commit messages
+- Commit messages should be clear and concise in describing the content of the commit
+- Commit messages should be written in English
```
Comment (bartvandebiezen) [inline git/README.md]: US English

Comment (7img) [reply git/README.md]: Changed! 👍

**Review by michaelmaat — APPROVED** [review]

File: git/README.md (hunk 1 above)
Comment (edgargressie) [reply git/README.md]: You changed it on the gitflow page, not in the README 😶

File: git/git-flow.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# The GIT flow
+
+![GIT flow](attachments/git-flow-aanzee)
```
Comment (edgargressie) [inline git/git-flow.md]: The path to the image isn't working. And I don't think we need to have a second resource for our Git flow. Not sure if I like our GitHub or Confluence for this part.

**Review by edgargressie — CHANGES_REQUESTED** [review]

File: git/README.md [hunk 3]
```diff
@@ -11,6 +11,11 @@ Prefix commit message with a [JIRA](https://www.atlassian.com/software/jira) ID
...
+- Commit messages should not include emojis
```
Comment (stevengunneweg) [inline git/README.md]: 'should' should be replaced with 'may'. 'should not ...' indicates that it is still allowed though not preferred. 'may not ...' disallows the use of emojis

**Review by stevengunneweg — CHANGES_REQUESTED** [review]

**Review by edgargressie — APPROVED** [review]

File: git/README.md [hunk 4]
```diff
@@ -4,8 +4,8 @@
 - Commit messages should be clear and concise in describing the content of the commit
 - Commit messages may not include emojis
 
-### JIRA ID
-Prefix commit message with a [JIRA](https://www.atlassian.com/software/jira) ID surrounded by square brackets when available. Always use the ID of the main task and ignore sub task ID's.
+### Jira ID
+Prefix commit message with a capitalized [Jira](https://www.atlassian.com/software/jira) ID surrounded by square brackets when available. Always use the ID of the main task and ignore sub task ID's.
```
Comment (michaelmaat) [inline git/README.md]: UPPERCASE insted of Capitalized ☺️

**Review by stevengunneweg — APPROVED** [review]

**Pull request merged**